.github/workflows/on-push.yaml

```diff
@@ -49,6 +49,12 @@ jobs:
             core.exportVariable('UI_LIB_PACKAGE_VERSION', ui_lib_package_version);
             core.exportVariable('UI_LIB_PACKAGE_NAME', `openshift-assisted-ui-lib-v${ui_lib_package_version}.tgz`)
 
+      - name: Validate that i18n locale files are updated and contains 'ai' prefix in keys
+        run: |
+          chmod +x ./scripts/validatei18n_files.sh
+          ./scripts/validatei18n_files.sh
+        shell: bash
+
```

Comment:
Not necessary to be performed on-push. A PR is mandatory to happen before a push event can occur. This only delays the CI as the number of translations grows!!!